ckanext/scheming/plugins.py

```diff
@@ -362,10 +364,12 @@ def _load_schema_module_path(url):
         m = __import__(module, fromlist=[''])
     except ImportError:
         return
-    p = os.path.join(os.path.dirname(inspect.getfile(m)), file_name)
-    if os.path.exists(p):
-        watch_file(p)
-        return loader.load(open(p))
+
+    path = os.path.join(os.path.dirname(inspect.getfile(m)), file_name)
+    if os.path.exists(path):
+        watch_file(path)
+        with open(path, 'rb') as schema_file:
+            return yaml.load(schema_file)
```

Comment:
I don't love parsing json with yaml because then users are going to sneak yamlisms into their schemas without renaming them. Also since pyyaml has code execution vulnerabilities this opens up sites with live schema editors to potential problems